packages/app-store/components.tsx

```diff
@@ -31,7 +31,7 @@ export const InstallAppButtonWithoutPlanCheck = (
           useDefaultComponent: true,
           disabled: props.disableInstall,
           onClick: () => {
-            mutation.mutate({ type: props.type });
+            mutation.mutate({ type: props.type, defaultInstall: true });
```

Comment:
This seems to be making any app that is being installed to be come the default.

So, if I install Google Meet, it becomes default and then if I install Zoom, it can then become default replacing Google Meet.

I think the intention is make an app default only through onboarding and this `InstallButton` component is used throughout the app.

Reply:
No. It is used inside InstallAppButton as well which is used elsewhere.